presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -214,6 +215,11 @@ public SystemSessionProperties(
                         PLAN_WITH_TABLE_NODE_PARTITIONING,
                         "Experimental: Adapt plan to pre-partitioned tables",
                         true,
+                        false),
+                booleanSessionProperty(
+                        COLOCATED_JOIN,
+                        "Experimental: Use a colocated join when possible",
+                        true,
```

Comment:
Do we want to make this enabled by default? If so, probably need to add a config property so it can be disabled if there are problems.